expressions/aggregation/AggregationConcreteHandle.hpp

```diff
@@ -27,10 +27,12 @@
 #include "catalog/CatalogTypedefs.hpp"
 #include "expressions/aggregation/AggregationHandle.hpp"
 #include "storage/HashTable.hpp"
+#include "storage/FastHashTable.hpp"
```

Comment:
A minor clarification, all the #include statements in the current block. You can leave out the #include glog block below as is. The #include block below is for third party includes.